js/main.js

```diff
@@ -489,7 +491,7 @@ function initiate() {
         }
 
         /* STAGE: check if math formulae are not split without necessity */
-        var badPos = latexString.search(/(\$|\\\))\s*(,|=|\+|-)?\s*(\$|\\\()/);
+        var badPos = latexString.search(/(\$(?!\$)|\\\)|\$\$)\s*(,|=|\+|-)?\s*(\$(?!\$)|\\\()/);
```

Comment:
The old regex would match `$$` as `$` then blank space then another `$`, and recognize this as an error.